src/Microsoft.TestPlatform.CoreUtilities/Helpers/FileHelper.cs

```diff
@@ -43,7 +43,7 @@ public bool DirectoryExists(string path)
         }
 
         /// <inheritdoc/>
-        public Stream GetStream(string filePath, FileMode mode, FileAccess access = FileAccess.ReadWrite)
+        public Stream GetStream(string filePath, FileMode mode, FileAccess access = FileAccess.Read)
```

Comment:
This function is used in other places too, Not just in DotnetTestHostManager.cs. Pass `access: FileAccess.Read` at [here](https://github.com/Microsoft/vstest/blob/master/src/Microsoft.TestPlatform.TestHostProvider/Hosting/DotnetTestHostManager.cs#L368) and [here](https://github.com/Microsoft/vstest/blob/master/src/Microsoft.TestPlatform.TestHostProvider/Hosting/DotnetTestHostManager.cs#L394)

Reply:
@smadala the default parameter is FileAccess.Read with this PR right? Why do we need to pass that in this way?

Reply:
We can't change the default value from ReadWrite to Read, Without changing it's references. I suggest we keep default value ReadWrite( `Opens a FileStream on the specified path with read/write access.` [More details](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.open?view=netframework-4.7.1#System_IO_File_Open_System_String_System_IO_FileMode_)). And just pass the access value Read in DotnetTestHostManager.cs